core/src/main/java/org/keycloak/representations/workflows/WorkflowRepresentation.java

```diff
@@ -195,8 +191,11 @@ public Builder withConfig(String key, List<String> values) {
         }
 
         public Builder name(String name) {
+            WorkflowRepresentation representation = new WorkflowRepresentation();
```

Comment:
Shouldn't the instantiation be moved to the constructor? Before the changes the `of` was called from line 33 and therefore the representation was always instantiated. 

With new version of the code it seems that if you do not call `name` as first in the line it may cause issues. I can fix that in follow up PR is preferred.

Reply:
Now it looks like `WorkflowRepresentation.withName("name")...`. Works for you?